frontend/src/stores/NavStore.js

```diff
@@ -0,0 +1,35 @@
+import {observable, action} from "mobx";
+import rootStore from "./RootStore";
+
+class NavStore {
+    @observable config = rootStore.mainStore.config;
+
+    @action
+    async downloadExcelReport() {
+        const excelUrl = this.config.editSettings.excelUrl;
+        await fetch(excelUrl, {
```

Comment:
this is the wrong way to go about this. This saves these file to succesufully to your JS store, but never sends them to the user. You can probably do something fancy in JS to save the blob to the user, but I'm changing it to let the browser handle it.